platforms/testplatform/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "mail": {
     "options": [{
-      "host": "mail",
+      "host": "localhost",
```

Comment:
here we just need to take care that other tests in travis still work as they require to have mail as hostname here. there exists a setting which should override it.

Reply:
we could optionally override the platform config with environment variables, e.g. MAIL_HOST and MAIL_PORT as in the redis configuration.

Reply:
thats's fine for now, we just have to check integration tests / docker setup that it still works

Reply:
this should be fixed now. we can override the host and port with MAIL_HOST and MAIL_PORT env variables. testplatform config is reverted to 'mail' as the host.